en/flight_modes_rover/index.md

```diff
@@ -45,6 +45,19 @@ Acro Mode is similar to Manual Mode, but with a closed-loop yaw rate control. In
 [Mission mode](../flight_modes/mission.md) is an automatic mode that causes the vehicle to execute a predefined autonomous [mission](../flying/missions.md) plan that has been uploaded to the flight controller.
 The mission is typically created and uploaded with a Ground Control Station (GCS) application, such as [QGroundControl](https://docs.qgroundcontrol.com/master/en/).
 
+Following is the list of currently implemented mission commands:
+
+| Command             | Description                                          |
+| ------------------- | ---------------------------------------------------- |
+| Mission start       | Starts the mission (necessary)                       |

```

Comment:
It is worth listing/linking to the MAVLink messages too. Why? Because that is ultimately what a mission is, and you can't know how the commands are being sent or named.

Reply:
Is [this](https://github.com/PX4/PX4-user_guide/blob/9cc703a2e440e4c11284f82b20ce3bf440f6663e/en/flight_modes_rover/index.md#mission-commands) what you had in mind?

Reply:
Perfect!